scripts/pre_commit_linter.py

```diff
@@ -1495,10 +1495,10 @@ def _match_line_breaks_in_controller_dependencies(all_files):
                 if stringfied_dependencies != function_parameters:
                     failed = True
                     print (
-                        '%s --> Please ensure that line breaks between '
-                        'the stringfied dependencies: "%s" and the function '
-                        'parameters: "%s" for the corresponding controller '
-                        'in this file exactly match.' % (
+                        'Please ensure that in %s file the line breaks pattern '
```

Comment:
I think the '%s -->' was better because it gives the developer an instant idea of the file where the error stands. In this case, they have to find the file name in this descriptive message. Also, this seems to be the convention followed by most of the linters like pylint, eslint etc as you might know.

Reply:
Okay, thanks.

Reply:
@anubhavsinha98, ah, I don't think that this message is much of an improvement over the earlier one. The keywords are still the same. We need an error message which explains the keywords such as `stringfied dependencies` more clearly and explicitly.

Reply:
Okay @apb7 will work on it.